scripts/component_readme/readme_template.md

```diff
@@ -1,31 +1,56 @@
 # {{ name }}
 
-### Description
+## Description
 {{ description }}
 
-### Inputs / outputs
+## Inputs / outputs
 
+### Consumes
 {% if consumes %}
 **This component consumes:**
-
```

Comment:
This newline is needed for formatting the list correctly.

Reply:
fixed